lib/Conversion/TorchToLinalg/Uncategorized.cpp

```diff
@@ -307,6 +308,90 @@ Value createDivModePayload(OpBuilder &b, Location loc,
   return quotient;
 }
 
+template <typename OpT>
+Value createRemainderPayload(OpBuilder &b, Location loc,
+                             const TypeConverter *converter,
+                             ValueRange payloadArgs, OpT op,
+                             ArrayRef<Value> operands) {
+  static_assert(std::is_same_v<OpT, AtenRemainderScalarOp> ||
+                    std::is_same_v<OpT, AtenRemainderTensorOp>,
+                "template type must be a tensor/scalar remainder");
```

Comment:
It seems that this causes a compile-time error. From what I understand, llvm::isa checks at runtime, and since it's in a static_assert, the change doesn't seem to work. I think it should then probably stay the same, unless there's another cleaner check that can be done at compile time.

The change right  below however:
```
  Value rhs = convertScalarToDtype(
      b, loc,
      std::is_same_v<OpT, AtenRemainderScalarOp> ? operands[1] : payloadArgs[1],
      dtype);
```

Can be modified with llvm::isa. Which I've done locally

Reply:
I hope you don't mind these kind of nit suggestions, I'm trying to learn the llvm api better.

Reply:
I've used is_one_of and isa in both situations in the newly pushed commit.